plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveConfig.java

```diff
@@ -1275,4 +1276,17 @@ public HiveConfig setPartitionProjectionEnabled(boolean enabledAthenaPartitionPr
         this.partitionProjectionEnabled = enabledAthenaPartitionProjection;
         return this;
     }
+
+    public boolean isBatchMetadataFetchEnabled()
```

Comment:
Since the core issue is specific to Thrift based metastore - can we move it to ThriftMetastore config ?

Reply:
@findepi WDYT?

Reply:
Makes sense.

Reply:
>  can we move it to ThriftMetastore config 

yes

Reply:
Moved to `ThriftMetastoreConfig`